hips/tiles/tile.py

```diff
@@ -189,25 +189,23 @@ def _from_raw_data(cls, meta: HipsTileMeta, raw_data: BytesIO) -> 'HipsTile':
                     data = hdu_list[0].data
                     header = hdu_list[0].header
             return cls(meta, data, header)
-        elif meta.file_format == 'jpg':
+        elif meta.file_format in {'jpg', 'png'}:
             with Image.open(raw_data) as image:
                 data = np.array(image)
             return cls(meta, data)
-        elif meta.file_format == 'png':
-            raise NotImplementedError()
         else:
-            raise ValueError(f'Tile file format not supported: {meta.file_format}. '
-                             'Supported formats: fits, jpg, png')
+            raise ValueError(f'Tile file format not supported: {self.meta.file_format}. '
+                              'Supported formats: fits, jpg, png')  # pragma: no cover
```

Comment:
Suggest to add a separate test actually triggering this ValueError and asserting that it comes out via `pytest.raises`, i.e. don't put `#pragma: no cover`